.gitignore

```diff
@@ -4,6 +4,7 @@ nurse_scheduling_problem.iml
 .gradle
 .idea
 .ipynb_checkpoints/
+quickrun.jl
```

Comment:
xD

Reply:
A to śmieszna historia xD
W skrócie do debugowania musiałem sobie jakieś ciągi funkcji wołać, a z faktu, że ponowny include w Julii nie reimportuje innych modułów, to uznałem krótki plik za najprostszą metodę. I to był nie pierwszy raz kiedy go potrzebowałem, więc uznałem, że go dodam może do gitignore, żeby móc klikać w spokoju add all xD

Reply:
https://github.com/timholy/Revise.jl